.github/workflows/update-cmo-jsonnet-deps.yaml

```diff
@@ -0,0 +1,58 @@
+name: Upgrade cmo jsonnet dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 1'
+env:
+  USER: 'github-actions[bot]<github-actions[bot]@users.noreply.github.com>'
+
+jobs:
+  jsonnet-update:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        repository: openshift/cluster-monitoring-operator
+        ref: master
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+    - name: update jsonnet dependencies
+      run: make update
+    - name: generate yaml assets
+      run: make generate
+    - name: get pr creation app token
+      id: pr
+      uses: getsentry/action-github-app-token@v1
+      with:
+        app_id: ${{ secrets.APP_ID }}
```

Comment:
Where are these variables coming from?

Reply:
These carries the credentials of the following GH apps, Let me put more details into the README :)

https://github.com/settings/apps/openshift-monitoring-bot
https://github.com/settings/apps/monitoring-commit-bot

Reply:
Can't we use the built-in `secrets.GITHUB_TOKEN` since it's short-lived and job specific?

Reply:
The transient GITHUB_TOKEN will be scoped only to the current repo. It can create branches and PR only with in the current repo scope but not outside. For our case, we need to create PR on repos in openshift org.